hogvm/python/stl/__init__.py

```diff
@@ -147,6 +147,14 @@ def replaceAll(name: str, args: list[Any], team: Optional["Team"], stdout: Optio
     return args[0].replace(args[1], args[2])
 
 
+def generateUUIDv4(
```

Comment:
Don't love the name tbh. Why not just `uuidv4()` or something like `uuid.v4()`. Feels a little overly verbose?

Reply:
https://clickhouse.com/docs/en/sql-reference/functions/uuid-functions#generateuuidv4